app-crypt/stegdetect/stegdetect-0.6.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="A Steganography Program"
+DESCRIPTION="Stegdetect is an automated tool for \
+detecting steganographic content in images."
+HOMEPAGE="https://github.com/abeluck/stegdetect"
+COPYRIGHT="2002 Niels Provos"
+LICENSE="BSD (4-clause)"
+REVISION="1"
+srcGitRev="28a4f074a71c682581314491224f9f41511c82e7"
+SOURCE_URI="https://github.com/abeluck/stegdetect/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="eb39bfcf50116c721c3bf0efaa168cb84b5141563b87d66235f1d21f3336e695"
+SOURCE_FILENAME="stegdetect-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="stegdetect-$srcGitRev"
+#PATCHES="stegdetect-$portVersion.patch"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	stegdetect = $portVersion
+	cmd:stegdetect = $portVersion
+	"
+REQUIRES=" 
+	haiku >= $haikuVersion
```

Comment:
Thanks @Sudoxo for this PR!
To show how we sometimes suggest changes, this is how we could present @extrowerk's comment:
```diff
 REQUIRES=" 
-	haiku >= $haikuVersion
+	haiku
 	lib:libjpeg
 	"
```
(Red = remove, Green = add)
Warning: the other lines start with a space but it should not be copied in case you copy/paste these lines.